man/labgrid-device-config.1

```diff
@@ -87,11 +103,39 @@ Path to the mxs\-usb\-loader binary
 .B \fBimx\-usb\-loader\fP
 Path to the imx\-usb\-loader binary
 .UNINDENT
+.SS TOOLS EXAMPLE
+.sp
+Configure the tool path for \fBimx\-usb\-loader\fP:
+.INDENT 0.0
+.INDENT 3.5
+.sp
+.nf
+.ft C
+tools:
+  imx\-usb\-loader: /opt/labgrid\-helper/imx\-usb\-loader
+.ft P
+.fi
+.UNINDENT
+.UNINDENT
 .SH IMPORTS
 .sp
 The \fBimports\fP key is a list of files or python modules which
 are imported by the environment after loading the configuration.
 Paths relative to the configuration file are also supported.
+.SS IMPORTS EXAMPLE
```

Comment:
We should point out this is mainly useful to load drivers and strategies, as they are imported before instantiating the target.